pkg/porter/schema.go

```diff
@@ -5,10 +5,12 @@ import (
 	"fmt"
 
 	"github.com/deislabs/porter/pkg/config"
-
 	"github.com/pkg/errors"
 )
 
+type jsonSchema = map[string]interface{}
```

Comment:
I am abusing type aliases just to make my tests easier to write/read. Am I a bad person? I can remove if this seems like a bad idea. I just got tired of writing out map[string]interface a hundred times. 😀

Reply:
I'm down w/ this; to me it definitely makes sense for comprehension.